src/lib/lmdb_storage/conv.ml

```diff
@@ -0,0 +1,46 @@
+open Core_kernel
+
+let uint32_be =
+  Lmdb.Conv.make
+    ~flags:
+      Lmdb.Conv.Flags.(
+        if Sys.big_endian && Int.equal Sys.int_size 4 then
```

Comment:
@georgeee I used Sys.int_size instead of is_int_size, since i cannot find it anywhere. I'm not sure if the `4` is right here. Sys.int_size is measuring int size in bits (so it will be 63 on majority of os). Is is_int_size expressed in bits too ? in that case 4 looks very suspicious